org.opent2t.sample.hub.superpopular/com.hue.hub/js/manifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <schemas>
+      <schema id="org.opent2t.sample.hub.superpopular" main="true" type="OCF" />
+      <schema id="oic.core" type="OCF" />
+      <schema id="oic.baseResource" type="OCF" />
+  </schemas>
+  <!-- associated onboarding module -->
+  <onboarding id="org.opent2t.onboarding.huehub">
```

Comment:
this should be the same name as the onboarding package, in this case org.opent2t.onboarding.hue